src/index.js

```diff
@@ -1,18 +1,17 @@
-let context
```

Comment:
Sorry, but the point of this repo is that I don't have to explicitly import all the examples files. I _want_ to use context here.